### PR TITLE
New naming standard CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by racerand**
I have updated CONTRIBUTING.md to include the new standard naming convention Inventory and Furniture. The rest of #505 I will be trying to recreate in a new pr.

### CODE REVIEW

**Comment by Mizipzor** [comment]
Cant we just replace the entire style guide section with: [StyleCop](https://github.com/TeamPorcupine/ProjectPorcupine/wiki/StyleCop). :)

**Comment by svmnotn** [comment]
@Mizipor does StyleCop look into xml files? if so we *NEED* that.

**Comment by Mizipzor** [comment]
It does not.

**Comment by Tranberry** [comment]
If you add to the ToC this little snippet I would be most pleased `[Adding Furniture and Inventory](#adding-furniture-and-inventory)  ` including space at the end so there will be a new line.

**Comment by svmnotn** [comment]
then, we can't just use style cop. Specially since this PR is about things in the xml files

**Comment by Mizipzor** [comment]
Well, I suppose xml style should remain there then. Although I though it was really one way people wrote it? Assuming we picked tabs or spaces.

**Comment by Mizipzor** [comment]
Ops didnt mean to close. :)

**Comment by Mizipzor** [comment]
@svmnotn Address @Tranberry's concern and Ill merge this.

**Comment by racerand** [comment]
Done @Mizipzor

**Comment by Mizipzor** [comment]
:)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -124,6 +125,22 @@ Additionally, for the sake of simplicity, we are standardizing on what should be
...
+
+We have standardized the objectTypes of Furniture and Inventory to match `type_material`, such as `wall_steel` and `generator_oxygen` or `generator_power`, for localization a matching prefix is added automaticly to objectType such as `inv` and `furn`. This means a few things:
+
+* When adding a new Furniture or inventory the files should have the objectType "type_material", you could give it the more english sounding name, as of now name is not used for anything.
+
+* For machines the convention will be `whatItDoes_whatItMakes`.
+
+* For multiword parts it will be `myType_myMaterial`.
+
+* In the Localization a line could be will be 'inv_type_material=Material Type' and `furn_type_material=Material Type`.
```
Comment (svmnotn) [inline CONTRIBUTING.md]: why are the inv ones not back ticks! grrr

**Pull request merged**